storage/postgresql/save_test.go

```diff
@@ -157,21 +152,16 @@ func TestSaveEventSql(t *testing.T) {
 				Content:   "test",
 				Sig:       "sig",
 			},
-			query: `INSERT INTO event (
-	id, pubkey, created_at, kind, tags, content, sig)
-	VALUES ($1, $2, $3, $4, $5, $6, $7)
-	ON CONFLICT (id) DO NOTHING`,
+			query:  `INSERT INTO event (id, pubkey, created_at, kind, tags, content, sig) VALUES ($1, $2, $3, $4, $5, $6, $7) ON CONFLICT (id) DO NOTHING`,
```

Comment:
why these changes? better leave it like it was, no?

Reply:
It could be. The force of habit of leaving everything in a single line.